apps/mobile/src/stacks/index.tsx

```diff
@@ -25,23 +33,40 @@ export const AppStack: FC<StackScreenProps<RootParamList>> = () => {
 	return (
 		<View style={styles.container} ref={modalContainerRef}>
 			<NavigationContainer ref={navigationRef} linking={linking} theme={theme}>
-				<Stack.Navigator screenOptions={screenOptions.navigator}>
-					<Stack.Screen
-						name="Splash"
-						component={SplashScreen}
-						options={screenOptions.fade}
-					/>
-					<Stack.Screen
-						name="Authentication"
-						component={AuthenticationStack}
-						options={screenOptions.fade}
-					/>
-					<Stack.Screen
-						name="Dashboard"
-						component={DashboardStack}
-						options={screenOptions.bottomFade}
-					/>
-				</Stack.Navigator>
+				{isLoggedIn ? (
+					<Drawer.Navigator
+						drawerContent={Sidebar}
+						screenOptions={{
+							headerShown: false,
+							drawerStyle: styles.drawer,
+							overlayColor: 'transparent',
+							drawerType: 'front',
+						}}
+						backBehavior="order"
+					>
+						<Drawer.Screen name="Dashboard" component={DashboardStack} />
+						{mockWidgets.map((widget) => (
+							<Drawer.Screen
+								key={widget._id}
+								name={widget._id}
+								component={WidgetScreen}
+							/>
+						))}
+					</Drawer.Navigator>
+				) : (
+					<Stack.Navigator screenOptions={screenOptions.navigator}>
+						<Stack.Screen
+							name="Splash"
+							component={SplashScreen}
+							options={screenOptions.fade}
+						/>
+						<Stack.Screen
+							name="Authentication"
+							component={AuthenticationStack}
+							options={screenOptions.fade}
+						/>
+					</Stack.Navigator>
+				)}
```

Comment:
DrawerStack did not override or conditionally rendered like this. Please keep the old navigation structure, and register Drawer inside `DashboardStack` -- only users who passed sign-in see `DashboardStack` together with the DrawerStack.